Source/WPF/MyMoney/Reports/NetWorthReport.cs

```diff
@@ -199,7 +202,7 @@ public override async Task Generate(IReportWriter writer)
                     writer.EndCell();
 
                     writer.StartCell();
-                    writer.WriteNumber(totalBalance.ToString("C"));
+                    writer.WriteNumber(this.myMoney.GetFormatedNormalizedAmount(totalBalance));

```

Comment:
There's 2 things going on here that I'd prefer to tease apart.  
1. The "conversion to a default currency" which I'd prefer to do in `GetNormalizedAmount`.  Today it's hard coded to USD, but could instead pick up a new new default currency property on the Money class. 
2. The "formatting to a locale" which you are adding here is an entirely new feature, that technically requires full globalization of the application, all strings from localized resources, all formatting of numbers from a localized NumberFormatInfo.  Doing it only for dollar amounts seems a bit weird.

Reply:
Now if you only have this currency code here so that you can put the write dollar symbol on things, and we are not opening the flood gates on full localization then I can understand that, but I think I'd then prefer the lookup to cultureCode to be done automatically with no user input, from an XML file or hard coded table - there's only so many international codes and they don't change that often... and if we do this we don't need to add it to the Currency table. The normal user wouldn't know anything about .net culture codes anyway...  The other thing we could try is search the "display name" of the culture and fuzzy match it with the "Name" column of the Currency, at a minimum auto-populate the culture code column that way.  Do you want thousands separators to change from commas to dots also? Which I think will also happen when you change CultureInfo...

Reply:
Agree, this is baby steps of the WIP

Reply:
The two functions are available 
```
       public string GetFormatedAmount(decimal amount, int dicimalPlace = 2)
        {
            return string.Format(this.CultureInfo, "{0:C" + dicimalPlace.ToString() + "}", amount);
        }

        public string GetFormatedNormalizedAmount(decimal amount, int dicimalPlace = 2)
        {
            amount /= this.rate;
            return this.GetFormatedAmount(amount, dicimalPlace);
        }
```


When using them in the report it felt cleaner to just call just one function instead of two

```
money.GetFormatedAmount(GetNormalizedAmount(123.45))
```

```
money.GetFormatedNormalizedAmount(123.45)
```